src/geo.c

```diff
@@ -176,8 +176,8 @@ int extractDistanceOrReply(client *c, robj **argv,
 int extractBoxOrReply(client *c, robj **argv, double *conversion,
                          double *height, double *width) {
```

Comment:
you had two more bugs, canceling one another.
```suggestion
int extractBoxOrReply(client *c, robj **argv, double *conversion,
                         double *width, double *height) {
```